BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -38,26 +38,21 @@ class StoresTableViewController: UITableViewController, SideMenuFilterDelegate {
     
     override func viewDidLoad() {
         super.viewDidLoad()
+
+        stores.removeAll()
```

Comment:
This isn't necessary here. When the Table View starts, this is gonna be empty already.